sdk/resourcemanager/Azure.ResourceManager/src/ArmClient.cs

```diff
@@ -49,9 +52,12 @@ public ArmClient(TokenCredential credential)
         /// </summary>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="options"> The client parameters to use in these operations. </param>
+        /// <param name="scope"> The scope to be included in acquired tokens. </param>
         /// <exception cref="ArgumentNullException"> If <see cref="TokenCredential"/> is null. </exception>
-        public ArmClient(TokenCredential credential, ArmClientOptions options)
```

Comment:
As Michael suggested, let's move the scope into ArmClientOptions.

Reply:
Done